### ISSUE TITLE
 undefined: nexus.HandlerErrorTypeDownstreamTimeout error

### ISSUE THREAD

**Issue by Flo4604** [temporalio/sdk-go#1687]
## Expected Behavior

To not see this error message.

## Actual Behavior
Getting the following error: 

```
# go.temporal.io/sdk/internal

../../../go/pkg/mod/go.temporal.io/sdk@v1.29.1/internal/internal_nexus_task_handler.go:480:36: undefined: nexus.HandlerErrorTypeDownstreamTimeout
```

## Specifications
  - Go Version: 1.23
  - Version: go.temporal.io/sdk v1.29.1
  - Platform: macOS 15.1 (24B83) / ubuntu github runner
  
  
 Using v1.29.0 fixes this issue.

**Comment by Flo4604** [comment]
Looks like this was fixed with the 1.30.0 updated.

**Comment by cretz** [comment]
Yes, this was an unfortunate case where our latest Go SDK version could not our latest Go API version. This should no longer be the case and we will try to prevent this issue in the future.
